Standards/scs-0001-v2-sovereign-cloud-standards.md

```diff
@@ -0,0 +1,349 @@
+
+The pull request MUST add exactly one SCS document,
+in the `Standards` folder.
+In the proposal phase,
+the document number MUST be replaced with `xxxx` in the file name.
+The major version MUST be 1.
+
+For a document with a slugified title `flavor-naming`,
+the file name would for instance be `scs-xxxx-v1-flavor-naming.md`.
+
+The metadata MUST indicate the intended `track` and `type` of the document,
+and the `status` MUST be set to `Proposal`.
+
+Upon acceptance by the group of people identified by the `track`,
+a number is assigned
+(the next unused number)
+and the proposer is asked
+to rename the file to replace the `xxxx` with that number.
+
+**Note:**
+Documents on the `Design Record` track MAY be proposed or accepted directly into `Stable` state,
+if no further discussion is required.
+
+Hereafter,
+the pull request can be merged
+and henceforth the document is an official SCS document in `Draft` state.
+
+#### Proposal of a major update to a stable document
+
+To propose major update to a Stable SCS document,
+a community participant creates a pull request on GitHub
+against the [standards repository in the SovereignCloudStack organisation][scs-standards-repo].
+
+The pull request MUST add exactly one SCS document,
+in the `Standards` folder.
+The document number MUST be the same as the document it is updating,
+and the major version number MUST be incremented by 1.
+The slugified title MAY be changed.
+
+It MUST refer to the old document in its `replaces` metadata.
+The pull request SHOULD NOT modify the previous document;
+deprecation of the previous document
+as well as adding the `replaced_by` metadata is a separate step,
+and can only be executed once the new document is `Stable`.
+
+For a document updating a hypothetical SCS-0390-v3 document,
+the file name may be `scs-0390-v4-flavor-naming-this-time-its-serious.md`.
+
+Other than the file naming difference,
+the proposal process is the same as for new documents.
+In particular, the new document starts out in `Draft` state
+and does not automatically become part of the normative corpus of an SCS release.
+
+### Development phase (Draft)
+
+In this phase,
+the document is developed inside the SCS community.
+
+It should not be considered to be normative for any SCS release,
+even if an SCS release happens after the acceptance of the document.
+
+Experimental and exploratory implementations are encouraged,
+however, implementors must be prepared for breaking changes.
+
+Changes to the documents are gated through pull requests.
+
+### Stabilized phase (Stable)
+
+Once the document is deemed ready for production use,
+its `status` is changed to `Stable`.
+
+After stabilization,
+changes to the document which may render existing implementations non-conformant
+MUST NOT be made.
+
+If a breaking change to an existing SCS document is deemed necessary,
+a new document with a new number shall be created
+and the old document SHOULD be deprecated.
+
```

Comment:
Yesterday in a meeting regards volume types between @mbuechse, @josephineSei  and @anjastrunk, we figured out, that conformance tests, be they automatic or manual, are mandatory prerequisites for transition to state stable. Past showed, conformance tests may need additional information, which are currently missing in standard and hard to add, if document is already stabilized. 

```suggestion

**Note:** Stabilization of standards requires the existence of conformance tests.
```

Reply:
This is the subject of this issue: https://github.com/SovereignCloudStack/standards/issues/502

Let's not mix multiple concerns.